src/components/BreweryListContainer.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { withRouter } from 'react-router-dom'
+import queryString from 'query-string'
+import BreweryList from './BreweryList'
+import Heading from './Heading'
+import Brewery from './Brewery'
+
+const filterResults = data =>
+  data.filter(
+    item => !item.name.toLowerCase().includes('brewery in planning')
+  )
+
+const fetchData = (searchParam, searchTerm) =>
```

Comment:
Initially it was because I didn't want this to be a blocking call when the component rendered.

I've since refactored this to use a hook that calls an async function for improved error handling, but I don't think async await adds anything over promises when inside the hook as it needs to be wrapped in a function that is immediately called so I've left it as a promise inside the hook.